### ISSUE TITLE
Fatal error: Cannot redeclare Google_Service_Safebrowsing_ThreatMatch::$threatType

### ISSUE THREAD

**Issue by bpolaszek** [google/google-api-php-client-services#27]
Looks like the `$threatType` property has been declared 2 times.

```php
# vendor/google/apiclient-services/Google/Service/Safebrowsing/ThreatMatch.php

class Google_Service_Safebrowsing_ThreatMatch extends Google_Model
{
  public $cacheDuration;
  public $platformType;
  protected $threatType = 'Google_Service_Safebrowsing_ThreatEntry'; //here
  protected $threatDataType = '';
  protected $threatEntryMetadataType = 'Google_Service_Safebrowsing_ThreatEntryMetadata';
  protected $threatEntryMetadataDataType = '';
  public $threatEntryType;
  public $threatType; // and there
}
```

Thanks,
Ben

**Comment by bshaffer** [comment]
Thanks for finding this! It will be fixed by #25

**Comment by bshaffer** [comment]
fixed in a2d43f4c7eace8c7aa16c5ea83191344c63b4526
